### ISSUE TITLE
LinkedIn "grant_type" is missing

### ISSUE THREAD

**Issue by kurucu** [inovector/mixpost#79]
### Detailed description

Hey all,

LinkedIn posting worked until recently, now I get the following error message. Was this me, Mixpost or a LinkedIn API change?

```
{ "error": "invalid_request", "error_description": "A required parameter \"grant_type\" is missing" }
```

**Comment by uhlhosting** [comment]
The closes to that I found is this: 

- https://forum.bubble.io/t/linkedin-api-a-required-parameter-grant-type-is-missing/300003 .

So I would recommend you to resync / relogin to your linkedIN again.

**Comment by kurucu** [comment]
@uhlhosting thank you, that gave me a head start.

So "updating" / refreshing didn't work. And a quirk (maybe of Mixpost?) is that I had to add the accounts again. However, I didn't want to delete them, and lose the history in e.g. analytics, so I just tried to add them again, and Mixpost magically updated the initial ones.

So you were right, and there's nothing really wrong with Mixpost.

But perhaps something can be done to streamline this, or make it clearer for the user? Especially to avoid deleting accounts or the implication that you might need to.
